client/src/client.ts

```diff
@@ -2498,6 +2505,17 @@ export abstract class BaseLanguageClient {
 		}
 	}
 
+	private logObjectTrace(data: any): void {
+		if (data.isLSPMessage && data.type) {
```

Comment:
@dbaeumer Not sure if this is what you meant by "if the data is an object then log it by JSON stringify here". Also not sure if that `isLSPMessage` is the best way to go. If you have better suggestions, let me know!

Reply:
No, in this case I would simply do 

```
this.outputChannel.append(`[LSP   - ${(new Date().toLocaleTimeString())}] `);
this.outputChannel.appendLine(`${JSON.stringify(data)}`);
```